### PR TITLE
Refactor ViewConfigStore and View3DStore

### LINKED ISSUE #121: Subsume the 3D view store into the view config store
The 3D view store is extraneous. It's purpose is to store the configuration for the 3D view, which should be handled by the view config store instead.

### PR DESCRIPTION
**Pull request by floryst**
This refactors the view configs and deprecates the view3D store.

Closes #121

### CODE REVIEW

**Comment by floryst** [comment]
@cjh1 @PaulHax this breaks up the view config store into smaller chunks. Everything is still under the view config store umbrella, but we are one step away from just making each one its own separate store. I'm okay keeping the view config store for now. What do you think?

Also, I tried to simplify all of the bookkeeping by introducing a new map object that is keyed off of two keys. I'm pretty sure it can be implemented using a class rather than vue reactivity utils, but that's just how I initially implemented it...

Still need to add the commit that will remove the view 3D store.

File: src/store/view-configs/camera.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const cameraConfigs = useDoubleRecord<CameraConfig>();
+
+  const removeView = (viewID: string) => cameraConfigs.delete1(viewID);
+  const removeData = (dataID: string, viewID?: string) => {
+    if (viewID) {
+      cameraConfigs.delete(viewID, dataID);
+    } else {
+      cameraConfigs.delete2(dataID);
+    }
+  };
```
[the remark this answers is not in the document]
Comment (floryst) [reply src/store/view-configs/camera.ts]: Pulled it out into common.

File: src/composables/useDoubleMap.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    Object.keys(map).forEach((k1) => _delete(k1, k2));
+  };
+
+  return {
+    set: _set,
+    get: _get,
+    has: _has,
+    delete: _delete,
+    delete1: _delete1,
+    delete2: _delete2,
```
[the remark this answers is not in the document]
Comment (floryst) [reply src/composables/useDoubleMap.ts]: Renamed to `deleteFirstKey` and `deleteSecondKey`.

File: src/store/view-configs/common.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const genSynViewConfigKey = (viewID: string, dataID: string) => {
+  return [viewID, dataID].join('|');
+};
```
[the remark this answers is not in the document]
Comment (floryst) [reply src/store/view-configs/common.ts]: Good catch, thanks!

**Review by cjh1 — APPROVED** [review]
@floryst This looks ready to go to me or is there more you would like todo?

**Comment by floryst** [comment]
Updated the double record with a test.

**Comment by floryst** [comment]
This is now ready for a final pass-through. The first commit contains the relevant changes.

File: src/store/view-configs/volume-coloring.ts [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+  useVolumetricScatteringBlending: boolean;
+  volumetricScatteringBlending: number;
+
+  useLocalAmbientOcclusion: boolean;
+  laoKernelSize: number;
+  laoKernelRadius: number;
+  ambient: number;
+  diffuse: number;
+  specular: number;
+}
```
Comment (cjh1) [inline src/store/view-configs/volume-coloring.ts:24]: This seems to be a duplicate of the type in `types/views.ts`?

Comment (floryst) [reply src/store/view-configs/volume-coloring.ts]: Thanks for catching that. I've removed the one in volume-coloring.ts.

**Pull request merged**